mod/ping.php

```diff
@@ -130,10 +130,10 @@ function ping_init(&$a) {
 		if ($a->config['register_policy'] == REGISTER_APPROVE && is_site_admin()){
 			$regs = q("SELECT `contact`.`name`, `contact`.`url`, `contact`.`micro`, `register`.`created`, COUNT(*) as `total` FROM `contact` RIGHT JOIN `register` ON `register`.`uid`=`contact`.`uid` WHERE `contact`.`self`=1");
 			if ($regs) {
-				$register = $regs[0]['total'];
+				$register_count = $regs[0]['total'];
 			}
 		} else {
-			$register = 0;
+			$register_count = 0;
```

Comment:
I know that it is already too late - but could you move this ````$register_count = 0;```` to the top? There is a possibility that the variable isn't defined. (When ````$regs```` is false)